tests/baselines/reference/tscWatch/programUpdates/when-changing-`allowImportingTsExtensions`-of-config-file.js

```diff
@@ -47,12 +47,7 @@ DirectoryWatcher:: Added:: WatchInfo: /user/username/projects/myproject/node_mod
 Elapsed:: *ms DirectoryWatcher:: Added:: WatchInfo: /user/username/projects/myproject/node_modules/@types 1 undefined Type roots
 DirectoryWatcher:: Added:: WatchInfo: /user/username/projects/node_modules/@types 1 undefined Type roots
 Elapsed:: *ms DirectoryWatcher:: Added:: WatchInfo: /user/username/projects/node_modules/@types 1 undefined Type roots
-[96mb.ts[0m:[93m1[0m:[93m8[0m - [91merror[0m[90m TS5097: [0mAn import path can only end with a '.ts' extension when 'allowImportingTsExtensions' is enabled.
-
-[7m1[0m import "./a.ts";
-[7m [0m [91m       ~~~~~~~~[0m
-
-[[90mHH:MM:SS AM[0m] Found 1 error. Watching for file changes.
```

Comment:
I think this error should be reported but currently it was reported here *accidentally*. It's not consistent with a regular - non-incremental/composite executions. I raised an issue about it here: https://github.com/microsoft/TypeScript/issues/58725

Reply:
FWIW resolving these would be https://github.com/microsoft/TypeScript/issues/50394